requirements/base.txt

```diff
@@ -17,3 +17,4 @@ metafone==0.5
 le-utils==0.0.9rc14
 kolibri_exercise_perseus_plugin==0.6.5
 jsonfield==2.0.1
+https://github.com/learningequality/morango/archive/master.zip#egg=morango
```

Comment:
can we specify a specific git commit? This is liable to produce unpredictable behavior